Fix AttributeError with np.longfloat and np.longcomplex in
python/test/test_fft.py.

```
python/test/test_fft.py:96: in <module>
    tol = {np.float32: 6e-7, np.float64: 2e-15, np.longfloat: 1e-18}
/usr/lib/python3.12/site-packages/numpy/__init__.py:400: in __getattr__
    raise AttributeError(
E   AttributeError: `np.longfloat` was removed in the NumPy 2.0 release. Use
`np.longdouble` instead.
```

```
python/test/test_fft.py:99: in <module>
    np.longdouble: np.longcomplex}
/usr/lib/python3.12/site-packages/numpy/__init__.py:400: in __getattr__
    raise AttributeError(
E   AttributeError: `np.longcomplex` was removed in the NumPy 2.0 release. Use
`np.clongdouble` instead
```

--- ducc-ducc0_0_34_0-origin/python/test/test_fft.py
+++ ducc-ducc0_0_34_0/python/test/test_fft.py
@@ -93,20 +93,20 @@
     return out
 
 
-tol = {np.float32: 6e-7, np.float64: 2e-15, np.longfloat: 1e-18}
+tol = {np.float32: 6e-7, np.float64: 2e-15, np.longdouble: 1e-18}
 ctype = {np.float32: np.complex64,
          np.float64: np.complex128,
-         np.longfloat: np.longcomplex}
+         np.longdouble: np.clongdouble}
 
 
 on_windows = ("microsoft" in platform.uname()[3].lower() or
               platform.system() == "Windows")
 on_arm = ("arm" in platform.machine().lower())
 on_ppc64le = ("ppc64le" in platform.machine().lower())
-true_long_double = (np.longfloat != np.float64 and not (on_windows or on_arm or on_ppc64le))
+true_long_double = (np.longdouble != np.float64 and not (on_windows or on_arm or on_ppc64le))
 dtypes = [np.float32, np.float64]
 if true_long_double:
-    dtypes += [np.longfloat]
+    dtypes += [np.longdouble]
 
 
 @pmp("len", len1D)
